getting-started.mdx

```diff
@@ -1,24 +1,148 @@
+- Click the `Coding` button to see the tools used for coding editing.
+  1. This section includes tools to write, run, and edit code.
+  2. We also have tools that support CSS and HTML.
+- Click the `Media` button to see the tools used for Media editing.
+  1. `Image` - We can add a picture from our desktop using this image tool.
+  2. `Video` - We can add a video from our desktop using this video tool.
+  3. `Youtube` - Just copy the URL of the Youtube video and paste it into the section.
+  4. `Draw` - You can draw shapes with the draw tool.
+  5. `Lottie block` - You can add animation to your module using the Lottie block.
+  6. `Audio` - Clicking audio lets you add audio and provides options for playing it.
+  7. `Cross line` - A horizontal line drawn through text or graphical elements to indicate removal, correction, or emphasis, It is commonly used in writing, design, and digital interfaces.
+- Click the `Course` button to see the tools used for Course editing.
+  1. `Math` - Using the math tool, we can easily add formulas and syntax to our module.
+  2. `Multiple-Choice` - We can create multiple-choice questions using the multiple-choice tool.
+- Click the `Apps` button to see the tools.
+  1. `ChatGPT` - Is an advanced AI language model developed by OpenAI. It is designed to engage in text-based conversations with users, offering answers, explanations, creative content, and more, using natural language understanding and generation. Click on ChatGPT and write your topic than click on generate.
+  2. `AL Image` - An AI image refers to images that are generated, modified, or enhanced using artificial intelligence (AI) techniques. Just click on Al Image write which kind of image you want and than click on generate.
+  3. `GIPHY` - Is an online platform and search engine that allows users to discover, create, and share GIFs (Graphics Interchange Format) — short, looping animated images.
+  4. `Elevenlabs` - Is an AI technology company specializing in text-to-speech and voice generation solutions. It is known for creating advanced AI models that can produce highly realistic and human-like speech from text input.
+  5. ` D-DAI` - Allows developers to create videos of Digital Humans using generative AI.
+
+## 10. Publishing modules and submitting.
+
+- Go to the [Course page](https://app.iclasser.com/teach).
+- Open your course.
+- In the curriculmn section, you will see your modules with three options:
+  1. `Edit content` - Used for editing your module.
+  2. `Delete` - You can delete your module.
+  3. `Publish` - Clicking the publish button submits your module to the course.
+- Once you have created all your course modules and published them, you are ready to send it for review.
+- Go to the `Management` section and click `Request Review` button.
+- Once you request a review, you will receive response by email; the process takes 48 hours.
+
+## 11. Creating instructor-led classes.

```

Comment:
👍, but the user should have verified their account before creating class. first go to your course `manage` section and verify your identity, by uploading documents. and then once reviewed the user will be verified to create classes.